src/codeu/chat/server/Model.java

```diff
@@ -67,10 +69,14 @@ public int compare(Time a, Time b) {
   private final Store<Time, Message> messageByTime = new Store<>(TIME_COMPARE);
   private final Store<String, Message> messageByText = new Store<>(STRING_COMPARE);
 
+  private HashMap<Uuid, InterestSet> interestMap = new HashMap<Uuid, InterestSet>(100);
+
   public void add(User user) {
     userById.insert(user.id, user);
     userByTime.insert(user.creation, user);
     userByText.insert(user.name, user);
+    interestMap.put(user.id, new InterestSet());
+    System.out.println("NEW SIZE: "+interestMap.size());
```

Comment:
There are a lot of print statements in this class. If you want to keep these print statements in here, it's a good idea to convert them to use the logger that's already part of the application (see `codeu.chat.util.Logger`). You should only do that if you think these messages will be useful to other developers. Otherwise, it might be a good idea to delete them.